superweather/superweather/LocationTool.m

```diff
@@ -0,0 +1,30 @@
+//
+//  LocationTool.m
+//  superweather
+//
+//  So far this only wraps CLLocationManager, but it could be useful
+//  if we do more with location.
+//
+//  Created by mikael petersson on 2016-04-17.
+//  Copyright © 2016 mikael petersson. All rights reserved.
+//
+
+#import "LocationTool.h"
+
+@implementation LocationTool
+
+-(LocationTool*) init {
```

Comment:
What do you think about using `instancetype` instead? See: http://nshipster.com/instancetype/